### PR TITLE
Require system.admin user for cluster get pair token

### LINKED ISSUE #922: GetPairToken should require system.admin user
**Is this a BUG REPORT or FEATURE REQUEST?**:
BUG

**What happened**:
* Any user can access system.admin tokens with GetPairToken

**What you expected to happen**:
* Only system.admin should be able to get cluster pair tokens when auth is enabled.

### PR DESCRIPTION
**Pull request by ggriffiths**
Signed-off-by: Grant Griffiths <grant@portworx.com>

**What this PR does / why we need it**:
* Requires system.admin user for cluster pair token

**Which issue(s) this PR fixes** (optional)
Closes #922

### CODE REVIEW

**Comment by ggriffiths** [comment]
Tested this locally with two tokens: `system.user` and `system.admin`.

System.user was denied with the error I've implemented.
System.admin returned a JWT System Token as expected.

File: pkg/role/sdkserviceapi.go [hunk 1]
```diff
@@ -74,8 +74,8 @@ var (
 					"credentials",
 					"objectstore",
 					"schedulepolicy",
-					"volume",
 					"mountattach",
+					"migrate",
```
Comment (jrivera-px) [inline pkg/role/sdkserviceapi.go]: Was this removed on purpose?

Comment (ggriffiths) [reply pkg/role/sdkserviceapi.go]: Yeah, github hides it, but that slice of strings actually has `"volume"` twice in there

Comment (ggriffiths) [reply pkg/role/sdkserviceapi.go]: we noticed it when looking through this code in the meeting. @lpabon said to remove it.

Comment (jrivera-px) [reply pkg/role/sdkserviceapi.go]: Yeah ok cool,  yeah I remember now.   Was just checking.

**Pull request merged**